CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ## [Unreleased]
 ### Added
 - localisation for the message a coupon is invalid
+- changed pipelines `shopgate.cart.updateProducts.v1` and `shopgate.cart.deleteProducts.v1` to meet specification in the Shopgate developer portal
```

Comment:
seems like you even have the verbiage `changed`, so it would not go under the `### Added` column :P

Reply:
Okay, misunderstood the title for something else. will add a `### Changed` title